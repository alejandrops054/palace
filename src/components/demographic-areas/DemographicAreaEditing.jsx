/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from "react";
import { Card, CardBody, CardTitle, Label, Col, Row, Input, Button } from 'reactstrap';
function DemographicAreaEditing() {
    return (
        <>
            <Card>
                <CardBody>
                    <CardTitle tag="h5">Demographic Area Editing</CardTitle>
                    <Row>
                        <Col md={6}>
                            <Label>Code</Label>
                            <Input type="text" />
                        </Col>
                        <Col md={6}>
                            <Label>Status</Label>
                            <Input type="select" name="select" id="exampleSelect">
                                <option>Alta</option>
                                <option>2</option>
                            </Input>
                        </Col>
                        <Col md={12}>
                            <Label>Name</Label>
                            <Input type="text" />
                        </Col>
                        <Col md={4}>
                            <Label>Amount</Label>
                            <Input type="text" />
                        </Col>
                        <Col md={4}>
                            <Label>Begining Date</Label>
                            <Input type="date" />
                        </Col>
                        <Col md={4}>
                            <Label>Ending Date</Label>
                            <Input type="date" />
                        </Col>
                        <Col md={12} className="justify-content-end">
                            <Button color="primary"><i className="fas fa-save"></i>Save</Button>
                            <Button color="primary"><i className="fas fa-times"></i>Cancel</Button>
                        </Col>
                    </Row>
                </CardBody>
            </Card>
        </>
    );
}

export default DemographicAreaEditing;
