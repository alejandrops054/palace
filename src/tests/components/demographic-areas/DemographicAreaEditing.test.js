/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import DemographicAreaEditing from '../../../components/demographic-areas/DemographicAreaEditing'

describe('DemographicAreaEditing component', () => {
    test('Debe mostrar snapshot', () => {

        const wrapper = shallow(<DemographicAreaEditing />)
        expect(wrapper).toMatchSnapshot();
    })

})