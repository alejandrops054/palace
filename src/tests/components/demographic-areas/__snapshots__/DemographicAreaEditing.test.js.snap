// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemographicAreaEditing component Debe mostrar snapshot 1`] = `
<Fragment>
  <Card
    tag="div"
  >
    <CardBody
      tag="div"
    >
      <CardTitle
        tag="h5"
      >
        Demographic Area Editing
      </CardTitle>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          md={6}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Code
          </Label>
          <Input
            type="text"
          />
        </Col>
        <Col
          md={6}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Status
          </Label>
          <Input
            id="exampleSelect"
            name="select"
            type="select"
          >
            <option>
              Alta
            </option>
            <option>
              2
            </option>
          </Input>
        </Col>
        <Col
          md={12}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Name
          </Label>
          <Input
            type="text"
          />
        </Col>
        <Col
          md={4}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Amount
          </Label>
          <Input
            type="text"
          />
        </Col>
        <Col
          md={4}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Begining Date
          </Label>
          <Input
            type="date"
          />
        </Col>
        <Col
          md={4}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Ending Date
          </Label>
          <Input
            type="date"
          />
        </Col>
        <Col
          className="justify-content-end"
          md={12}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Button
            color="primary"
            tag="button"
          >
            <i
              className="fas fa-save"
            />
            Save
          </Button>
          <Button
            color="primary"
            tag="button"
          >
            <i
              className="fas fa-times"
            />
            Cancel
          </Button>
        </Col>
      </Row>
    </CardBody>
  </Card>
</Fragment>
`;
